packages/expo-notifications/ios/EXNotifications/Notifications/Scheduling/EXNotificationSchedulerModule.m

```diff
@@ -75,12 +71,31 @@ - (void)setModuleRegistry:(UMModuleRegistry *)moduleRegistry
 UM_EXPORT_METHOD_AS(cancelScheduledNotificationAsync,
                      cancelNotification:(NSString *)identifier resolve:(UMPromiseResolveBlock)resolve rejecting:(UMPromiseRejectBlock)reject)
 {
-  [[UNUserNotificationCenter currentNotificationCenter] removePendingNotificationRequestsWithIdentifiers:@[identifier]];
-  resolve(nil);
+  [self cancelScheduledNotificationAsync:identifier resolve:resolve rejecting:reject];
```

Comment:
If I'm not mistaken it shouldn't be required to add this middle method only to be able to override the method in a subclass—`UM_EXPORT_METHOD_AS` was designed to handle this well, it creates a native method with the same signature as the second argument to this macro, so here you'd get:
```objc
- (void)cancelNotification:(NSString *)identifier resolve:(UMPromiseResolveBlock)resolve rejecting:(UMPromiseRejectBlock)reject;
```
You can use this method to override the behavior. Xcode even suggests to use these methods!

![Zrzut ekranu 2020-05-20 o 19 22 27](https://user-images.githubusercontent.com/1151041/82477110-408b4c00-9acf-11ea-9543-edcbec71ad9f.png)